File: chpt1/section_1_1/problem_1_1_1.m
t = -3.0:0.01:3.0;

y1=exp(t);
y2=exp(1)*t;

plot(t,y1,t, y2);
legend("y1(t)=e^{t}", "y2(t)=e*t")

grid on;
title("plot of y1(t)=e^{t} and y2(t)=e*t");
xlabel("time");
ylabel("y");

[rows,cols] = size(t);
for i=1:cols
    if(y1(1,i) == y2(1,i))
        result = "when t = " + t(1,i) + ", y1 = " + y1(1,i) + " and y2 = " + y2(1,i);
        disp(result)
    end
end

File: chpt1/section_1_1/problem_1_1_1.py
import numpy as np
import matplotlib.pyplot as plt

# Generate t values
t = np.linspace(-2, 2, 400)  # 400 points from -2 to 2
y1 = np.exp(t)  # Compute e^t for each t
y2 = np.e * t

# Create the plot
# 8 high and 4 wide
plt.figure(figsize=(8, 4))
plt.plot(t, y1, label='$y1 = e^t$')
plt.plot(t, y2, label='$y2 = e*t$')

# Adding title and labels
plt.title('Graph of $y1 = e^t$ and $y2=e*t$')
plt.xlabel('t')
plt.ylabel('y')

# Add a grid and legend
plt.grid(True)
plt.legend()

# Show the plot
plt.show()